rust-toolchain

```diff
@@ -1 +1,2 @@
-1.63.0
+[toolchain]
+channel = "nightly-2022-10-03"
```

Comment:
What prompted this? I really want to use a stable toolchain if possible. If this is coming from some dependency, I will go and pressure them to use a stable too

Reply:
It's probably this 
https://github.com/filecoin-project/rust-fil-proofs#building-for-arm64

Which I can solve locally by cross-compiling.